util/Migrator/DbScripts/2024-01-03_00_FixGrantSave.sql

```diff
@@ -1,61 +1,54 @@
-CREATE OR ALTER PROCEDURE [dbo].[Grant_Save]
-    @Key  NVARCHAR(200),
-    @Type  NVARCHAR(50),
-    @SubjectId NVARCHAR(200),
-    @SessionId NVARCHAR(100),
-    @ClientId NVARCHAR(200),
-    @Description NVARCHAR(200),
-    @CreationDate DATETIME2,
-    @ExpirationDate DATETIME2,
-    @ConsumedDate DATETIME2,
-    @Data NVARCHAR(MAX)
+CREATE OR ALTER PROCEDURE [dbo].[Grant_Save] @Key NVARCHAR(200),
```

Comment:
⛏️ Newline for the first parameter -- agree?

Reply:
Yes, resubmitted